CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -561,100 +561,69 @@
+                                    <MenuItem Header="Disable" Command="{Binding SelectedJobSchedule.DisableJobSchedule}" CommandParameter="{Binding SelectedJobSchedule}" />
+                                    <MenuItem Header="Delete" Command="{Binding SelectedJobSchedule.Delete}" CommandParameter="{Binding SelectedJobSchedule}" />
                                 </ContextMenu>
                             </DataGrid.ContextMenu>
                             <DataGrid.Columns>
-                                <DataGridTextColumn Header="Id" Binding="{Binding Id}"/>
-                                <DataGridTextColumn Header="Display Name" Binding="{Binding DisplayName}"/>
+                                <DataGridTextColumn Header="Id" Binding="{Binding Id}" />
+                                <DataGridTextColumn Header="Display Name" Binding="{Binding DisplayName}" />
                                 <DataGridTextColumn Header="State" Binding="{Binding State}">
                                     <DataGridTextColumn.CellStyle>
                                         <Style TargetType="DataGridCell">
-                                            <Setter Property="Foreground" Value="{Binding State, Converter={StaticResource JobScheduleStateColorConverter}}"/>
-                                            <Setter Property="FontWeight" Value="Bold"/>
-                                            <Setter Property="Background" Value="White"/>
+                                            <Setter Property="Foreground" Value="{Binding State, Converter={StaticResource JobScheduleStateColorConverter}}" />
+                                            <Setter Property="FontWeight" Value="Bold" />
+                                            <Setter Property="Background" Value="White" />
                                         </Style>
                                     </DataGridTextColumn.CellStyle>
                                 </DataGridTextColumn>
-
-                                <DataGridTextColumn Header="" Width="*" CanUserReorder="False" CanUserSort="False"/>
+                                <DataGridTextColumn Header="" Width="*" CanUserReorder="False" CanUserSort="False" />
                             </DataGrid.Columns>
                         </DataGrid>
                     </Grid>
-
                 </TabItem>
                 <TabItem Header="{Binding JobTabTitle}" IsSelected="{Binding JobTabIsSelected}">
                     <Grid>
                         <Grid.RowDefinitions>
                             <RowDefinition Height="Auto"/>
                             <RowDefinition Height="*"/>
                         </Grid.RowDefinitions>
-                        <StackPanel
-                            Grid.Row="0"
-                            Orientation="Horizontal">
+                        <DockPanel LastChildFill="True">
                             <Button

```

Comment:
Yes, the StackPanel is a pain in this regard - it only takes up as much space as it thinks it needs. I usually fall back to using a Grid with columns where the last column width is set to "*", but that is similar to a DockPanel, so I'm happy keeping that as-is.